Sources/SimplyCoreAudio/Public/Enums/Scope.swift

```diff
@@ -28,12 +28,16 @@ public enum Scope {
     /// side of an object.
     case playthrough
 
-    /// The AudioObjectPropertyElement value for properties that apply to the master
-    /// element or to the entire scope.
+    /// The AudioObjectPropertyElement value for properties that apply to the main
+    /// element or to the entire scope. Using deprecated naming
```

Comment:
I don't believe `@available` is valid here, hence why I didn't add it; but I could be wrong